### DISCUSSION TITLE
Column mode selection and edition

### DISCUSSION THREAD

**Discussion by ced75** [dbeaver/dbeaver#334] [Q&A] [answered]
Eclipse supports column mode selection and edition, which is very useful for particular SQL synthaxes (when writting CASE..., multi joins, etc.).
Is it possible to implement this column mode in DBeaver editor ?

Thanks in advance.

**Comment by serge-rider** [comment]
It is already there.
As in Eclipse: Alt+Shift+A

**Reply by netdragonboberb** [reply]
Thanks. I had the same issue and this worked (albeit it froze up dbeaver for a while when I added a comma for the entire selection). My only feedback is discoverability isn't great as shown by the person asking the question. Many editors (such as Notepad++ and Microsoft SQL) support ALT+SHIFT+up/down, so that's what I believe most people will be trying. Someday, it'd be nice to provide that as an alternative, though I do understand that is a feature request, not something to be addressed for this ticket.

Here is a screenshot from Notepad++ after pressing ALT+UP a few times in a long list of ids and then typing "comma"
![image](https://user-images.githubusercontent.com/4391690/120829698-b2dc9c00-c52b-11eb-828a-7232b2e790ce.png)

**Comment by ced75** [comment, marked as answer]
Great! I didn't find it.
Thanks for your help and for DBeaver.
